File: game/src/main/java/de/d2dev/fourseasons/prototypes/lua/FirstLua.lua
for i=1,5 do
	print("Hallo!!!")
end

i = 10.5
print(i)

function myprint ()
	print(11)
end

myprint()

function boundPrint ()
	bound = luajava.newInstance( "de.d2dev.fourseasons.prototypes.lua.BindMe" )
	bound:printOnConsole()
end

boundPrint()
boundPrint()

File: game/src/main/java/de/d2dev/fourseasons/prototypes/lua/FirstLua.java
package de.d2dev.fourseasons.prototypes.lua;

import org.luaj.vm2.LuaValue;
import org.luaj.vm2.lib.jse.JsePlatform;

public class FirstLua {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		String script = FirstLua.class.getResource("FirstLua.lua").getPath();
		
		// create an environment to run in
		LuaValue _G = JsePlatform.standardGlobals();
		_G.get("dofile").call( LuaValue.valueOf(script) );
	}
}
